composer.json

```diff
@@ -1,14 +1,14 @@
 {
-    "name": "bitpesa/bitpesa-php-sdk",
-    "version": "0.3.0",
-    "description": "BitPesa Client Library for PHP",
-    "homepage": "http://api.bitpesa.co/documentation",
+    "name": "transferzero/transferzero-php-sdk",
+    "version": "1.0.0",
+    "description": "TransferZero Client Library for PHP",
+    "homepage": "http://api.transferzero.com/documentation",
     "license": "MIT",
     "authors": [
         {
             "name": "BTC Africa S.A",
```

Comment:
Are we leaving this in? Or should we be changing this to Transfer Zero as well?

Reply:
good catch